Contest_API.md

```diff
@@ -760,6 +762,18 @@ can only be non-null when start time is null.
 The request should fail with a 403 error code if the contest is started or within 30s of
 starting, or if the new start time is in the past or within 30s.
 
+Clients with the `contest_thaw` [capability](#capabilities) have the ability to
+set a time when the contest will be thawed via a PATCH method.
+
+The Patch must include a valid JSON object with exactly three properties:
+the contest `id` (used for verification) and a `contest_thaw_time`, which can
+either be a `<TIME>` value or the specialstring value `"now"`, which indicates
```

Comment:
Typo: special string

I'm not sure I love 'now' vs setting the current time, but I'm ok with it. FWIW the CDS has similar extension for start time where you can essentially say "start in 5m" or "reduce the pause time by 30s" to make clients life easier, and I waffle between thinking 'it should be removed' and 'it makes things so much easier for clients/users'.

Reply:
I’m struggling with this as well. But I van imagine not all CCSes can start at any time, and can only do now. If we think we all can do any time I can drop it

Reply:
Fair enough. If more CCSs are likely to implement 'now' but not timed then let's go with that.

Reply:
Yeah. DOMjudge can do 'any' time, so if the other big CCS'es can also do it, I'm fine dropping `now`.

Reply:
Hmmm, but what should happen if the client sends a thaw time that is a few seconds in the past, e.g. due to network delay? In principle, I think that this is (or could be considered) invalid, because the CCS may not be able to retroactive thaw the contest at a past time. So from that point of view `now` seems a good option...

Reply:
I kinda left that in the middle on purpose by saying that an invalid time should give 403. But I do agree thawing in the past should not be possible. So yeah, maybe you are right and we should keep `now` if you want to thaw... now.